src/htmx.js

```diff
@@ -3008,26 +3014,32 @@ return (function () {
             var splitPath = path.split("#");
             var pathNoAnchor = splitPath[0];
             var anchor = splitPath[1];
-            var finalPathForGet = null;
-            if (verb === 'get' || verb === 'delete') {
-                finalPathForGet = pathNoAnchor;
+
+            // Override the useUrlParams config if an extension defines encodeParameters
+            withExtensions(elt, function(extension) {
+                if (typeof extension.encodeParameters === 'function') {
+                    useUrlParams = false
+                }
+            })
+
```

Comment:
think we can drop this now?  I don't like signaling this via the presence of a function, would rather the extension do so via the `htmx:configRequest` event if necessary...

Reply:
Yup that's smart, done.